benchmark/Streamly/Benchmark/Data/Stream/StreamK.hs

```diff
@@ -442,6 +543,9 @@ filterAllInNested str = runStream $ do
 -- Nested Composition Pure lists
 -------------------------------------------------------------------------------
 
+-- XXX We should remove this? Or move this elsewhere. We have streaming
+-- benchmarks for comparision I guess.
```

Comment:
There are several list benchmarks here for comparison with lists. It is easy and convenient to see the comparisons when they are here, otherwise you will have to add a separate module for list benchmarks with the same names and then add a comparison in bench.sh.

Reply:
Guess we can keep this for now.

Reply:
I'm keeping the XXX comment though.